### PR TITLE
tracing: bump cpp2sky v0.2.0

### PR DESCRIPTION
**Pull request by Shikugawa**
Signed-off-by: Shikugawa <rei@tetrate.io>

For an explanation of how to fill out the fields, please see the relevant section
in [PULL_REQUESTS.md](https://github.com/envoyproxy/envoy/blob/main/PULL_REQUESTS.md)

Commit Message:
Additional Description:
Risk Level:
Testing:
Docs Changes:
Release Notes:
Platform Specific Features:
[Optional Runtime guard:]
[Optional Fixes #Issue]
[Optional Deprecated:]
[Optional [API Considerations](https://github.com/envoyproxy/envoy/blob/main/api/review_checklist.md):]

### CODE REVIEW

File: bazel/repository_locations.bzl [hunk 1]
```diff
@@ -322,26 +322,26 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         project_desc = "Data Collect Protocols of Apache SkyWalking",
         project_url = "https://github.com/apache/skywalking-data-collect-protocol",
         name = "skywalking_data_collect_protocol",
-        sha256 = "fa9ac679624217f30b6e8d5c450365386c610e2d08188a20f0340c3b14401833",
-        urls = ["https://github.com/apache/skywalking-data-collect-protocol/archive/v8.3.0.zip"],
-        strip_prefix = "skywalking-data-collect-protocol-8.3.0",
-        version = "8.3.0",
+        sha256 = "4a3401098e0e49ae56d03cc4890b21c75bab6ccf1c157d73bb0c93fc15bb07c7",
+        urls = ["https://github.com/apache/skywalking-data-collect-protocol/archive/{version}.tar.gz"],
```
Comment (moderation) [inline bazel/repository_locations.bzl:326]: Use latest release here? https://github.com/apache/skywalking-data-collect-protocol/releases/tag/v8.4.0

File: bazel/repository_locations.bzl [hunk 2]
```diff
@@ -322,26 +322,26 @@ REPOSITORY_LOCATIONS_SPEC = dict(
...
         cpe = "N/A",
     ),
     com_github_skyapm_cpp2sky = dict(
         project_name = "cpp2sky",
         project_desc = "C++ SDK for Apache SkyWalking",
         project_url = "https://github.com/SkyAPM/cpp2sky",
-        sha256 = "a8d870bb4b1c4a05eae319f689d1948927f3f0a5b5fe524db73a4c04121a339a",
-        version = "0.1.1",
+        sha256 = "7122b8e2ffee33d16998e767d7971906aa88826c16685842ab8a93378cd910b0",
+        version = "a4971da76507c0cb8199071c212867e1d2558c0c",
```
Comment (moderation) [inline bazel/repository_locations.bzl:339]: 0.2.0 not release yet? https://github.com/SkyAPM/cpp2sky/releases

File: bazel/repository_locations.bzl (hunk 1 above)
Comment (Shikugawa) [reply bazel/repository_locations.bzl:326]: The latest version of cpp2sky implements feature that doesn't be implemented in v8.4.0.

File: bazel/repository_locations.bzl (hunk 2 above)
Comment (Shikugawa) [reply bazel/repository_locations.bzl:339]: Once I released v0.2.0 but several problems had occurred while Envoy integration. So we will release it again after pending PR had merged, then ready to merge this.

**Comment by Shikugawa** [comment]
@lizan PTAL

**Comment by Shikugawa** [comment]
@wbpcode I'm very confused how is clear way to this problem (described [here](https://github.com/envoyproxy/envoy/pull/15333/files#diff-f12e5130222c34559664a404db8cef1b3c8217a7be0ae1ee65c23a2eb3d65afdR52-R60) for more detail). Do you have any idea about this?

**Pull request closed without merging**